internal/component/mimir/rules/kubernetes/events.go

```diff
@@ -185,6 +184,16 @@ func (e *eventProcessor) desiredStateFromKubernetes() (kubernetes.RuleGroupsByNa
 				return nil, fmt.Errorf("failed to convert rule group: %w", err)
 			}
 
+			var sourceTenants []string
+			if rule.Annotations[AnnotationsSourceTenants] != "" {
+				sourceTenants = regexp.MustCompile(`\s*,\s*`).
```

Comment:
No need to recompile the same regexp here in a loop, move to package level variable so it can be reused

Reply:
This is fixed, thank you